src/algebra/jordan/basic.lean

```diff
@@ -0,0 +1,230 @@
+/-
+Copyright (c) 2021 Christopher Hoskin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Christopher Hoskin
+-/
+import algebra.ring.basic
+import algebra.lie.of_associative
+import data.real.basic
+import linear_algebra.basic
+
+/-!
+# Jordan algebras
+
+We define a set of conditions, formulated in terms of commuting multiplication operators, for the
+multiplication on a non-unital, non-associative semiring to be a Jordan multiplication. When the
+multiplication is commutative, these take a particularly simple form.
+
+A real Jordan algebra `A` can be introduced by
+```
+variables {A : Type*} [non_unital_non_assoc_ring A] [module ℝ A] [smul_comm_class ℝ A A]
+  [is_scalar_tower ℝ A A] [comm_jordan A]
+```
+
+## Main results
+
+- lin_jordan : Liniarisation of the commutative Jordan axiom
```

Comment:
```suggestion
- lin_jordan : Linearization of the commutative Jordan axiom
```

Reply:
Does Mathlib use Oxford spelling? https://en.wikipedia.org/wiki/Oxford_spelling

Reply:
Not sure, but whichever it is it ought to start with "linear".  Neither word appears in mathlib yet, and all I know is that "linearization" is more common than "linearisation" according to [Google Ngram Viewer](https://books.google.com/ngrams/graph?content=linearisation%2Clinearization&year_start=1800&year_end=2019&corpus=26&smoothing=3&direct_url=t1%3B%2Clinearisation%3B%2Cc0%3B.t1%3B%2Clinearization%3B%2Cc0)